### PR TITLE
DDP-4807-add-script-to-retreive-address-pluscode

### PR DESCRIPTION
**Pull request by Rinchan20**
Have an ops request to change an address - one of the the columns in a mail address is for the pluscode. This is just to add a script to enable quick use/retrieval of pluscode that has the same precision we already use. I haven't added a test for it since OLCServiceTest seems to take care of testing precision in general.

## Context

_Say a few words about the bigger picture beyond this PR. How do your changes improve life for our users? Link directly to the jira ticket (just type "DDP-..." and it'll automatically link) to help your teammates quickly get up to speed on the motivation for these changes._

## Checklist

- [X] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [X] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [X] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [X] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/src/main/java/org/broadinstitute/ddp/AddressPlusCode.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.ddp;
+
+import com.typesafe.config.Config;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.broadinstitute.ddp.service.OLCService;
+import org.broadinstitute.ddp.util.ConfigManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AddressPlusCode {
```
Comment (yufengwng) [inline pepper-apis/src/main/java/org/broadinstitute/ddp/AddressPlusCode.java]: Does this have to live under `src/main`? I'm a bit concerned that folks might get confused and start using it in real code. How about we move to this `src/test`? Maybe just encapsulate this as a `@Test` in `OLCServiceTest` or another test class?

Comment (yufengwng) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/AddressPlusCode.java]: It sounds like we just want to run this locally every once in a while right? This should probably do the trick:
```java
public class OLCServiceTest ... {
  ...

  // Uncomment to run this!
  // @Test
  public void computePluscode() {
    ...
  }
}

Comment (Rinchan20) [reply pepper-apis/src/main/java/org/broadinstitute/ddp/AddressPlusCode.java]: It doesn't need to live under `src/main`, `src/test` sounds good - I was not thinking of that concern, I'll make sure to keep it in mind in the future.

I like the solution proposed! I'll do it that way.

**Review by yufengwng — APPROVED** [review]
👍 Looks good!

**Review by zyme — COMMENTED** [review]
As  long as we're not running this at a utility when pointed at production, this is okay, but it  would be safer for this to run as  a  `main()` instead of a test so that it does not apply database changes via liquibase which:
1) are not needed for this test
2) if run on a non-prod branch while the config points at production, would apply database changes to prod as a side effect, which would put production in an inconsistent and potentially unusable state

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**